extensions/raindrop/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "search_raindrop",
+  "title": "Search Raindrop",
```

Comment:
I'd propose you choose something different here because this name will be shown in the store and it's more clear to users in the store at least what service is this. I would change to something like **Raindrop.io Bookmarks** or something like that.